### PR TITLE
Add parameter to configure use of tf static

### LINKED ISSUE #295: Allow configuring use tf_static (or not) for static transforms
Many nodes and libraries in the ROS ecosystem still don't support tf_static (TF2). This makes it difficult (or impossible) to use topics published in frames whose TF chain is published through tf_static (i.e.: 'laser').

Please provide a configuration option such as `use_tf_static` to enable or disable the use of tf_static for the publishing of these transforms, such as it is provided in the robot_state_publisher: http://wiki.ros.org/robot_state_publisher#Parameters

### PR DESCRIPTION
**Pull request by PerrineAguiar**
Fixes #295.

### CODE REVIEW

**Comment by PerrineAguiar** [comment]
FYI @adamantivm

**Comment by PerrineAguiar** [comment]
@mcopejans ptal

File: tango_ros_common/tango_ros_native/src/tango_ros_node.cpp [hunk 1]
```diff
@@ -578,6 +582,7 @@ void TangoRosNode::TangoDisconnect() {
 }
 
 void TangoRosNode::PublishStaticTransforms() {
+  std::vector<geometry_msgs::TransformStamped> tf_transforms;
```
Comment (mcopejans) [inline tango_ros_common/tango_ros_native/src/tango_ros_node.cpp]: Nit. Add `tf_transforms.reserve(NUMBER_OF_STATIC_TRANSFORMS);`

**Comment by PerrineAguiar** [comment]
@mcopejans Thanks for your review.

**Pull request merged**